js_modules/dagit/packages/core/src/runs/MetadataEntry.tsx

```diff
@@ -191,6 +190,14 @@ export const METADATA_ENTRY_FRAGMENT = gql`
   }
 `;
 
+const IconButton = styled.button`
```

Comment:
I wonder if this should be moved to Icon.tsx as an additional `export`? It seems like we've got it in a few places here.

Reply:
Definitely possible. I think this is worth looking at as a followup.